README.md

```diff
@@ -275,6 +279,9 @@ learning more about html.  I suggest starting with their
 
 Another useful resource is [Mozilla Developer Network](http://developer.mozilla.org/).
 
+
+>>>PAGE 3<<<
```

Comment:
@brian-dawn This might be a good place for introducing the project structure and moving the readme section introduced [here](https://github.com/clojurebridge-minneapolis/track1-chatter/pull/8/files#diff-13a10e9e71c99cd3c650505eccaa1c8aR46).